src/transformers/models/t5la/convert_t5lax_checkpoint_to_flax.py

```diff
@@ -0,0 +1,237 @@
+            "kernel"
+        ]
+        t5lax_enc_dec_attention_query = t5lax_model["target"]["decoder"][layer_name]["encoder_decoder_attention"][
+            "query"
+        ]["kernel"]
+        t5lax_enc_dec_attention_value = t5lax_model["target"]["decoder"][layer_name]["encoder_decoder_attention"][
+            "value"
+        ]["kernel"]
+
+        # Layer Normalization
+        t5lax_cross_layer_norm = t5lax_model["target"]["decoder"][layer_name]["pre_cross_attention_layer_norm"][
+            "scale"
+        ]
+
+        # MLP
+        if split_mlp_wi:
+            t5lax_mlp_wi_0 = t5lax_model["target"]["decoder"][layer_name]["mlp"]["wi_0"]["kernel"]
+            t5lax_mlp_wi_1 = t5lax_model["target"]["decoder"][layer_name]["mlp"]["wi_1"]["kernel"]
+        else:
+            t5lax_mlp_wi = t5lax_model["target"]["decoder"][layer_name]["mlp"]["wi"]["kernel"]
+
+        t5lax_mlp_wo = t5lax_model["target"]["decoder"][layer_name]["mlp"]["wo"]["kernel"]
+
+        # Layer Normalization
+        tx5_mlp_layer_norm = t5lax_model["target"]["decoder"][layer_name]["pre_mlp_layer_norm"]["scale"]
+
+        # Assigning
+        flax_model.params["decoder"]["block"][str(layer_index)]["layer"]["0"]["SelfAttention"]["k"]["kernel"] = (
+            t5lax_attention_key
+        )
+        flax_model.params["decoder"]["block"][str(layer_index)]["layer"]["0"]["SelfAttention"]["o"]["kernel"] = (
+            t5lax_attention_out
+        )
+        flax_model.params["decoder"]["block"][str(layer_index)]["layer"]["0"]["SelfAttention"]["q"]["kernel"] = (
+            t5lax_attention_query
+        )
+        flax_model.params["decoder"]["block"][str(layer_index)]["layer"]["0"]["SelfAttention"]["v"]["kernel"] = (
+            t5lax_attention_value
+        )
+
+        flax_model.params["decoder"]["block"][str(layer_index)]["layer"]["0"]["layer_norm"]["weight"] = (
+            t5lax_pre_attention_layer_norm
+        )
+
+        flax_model.params["decoder"]["block"][str(layer_index)]["layer"]["1"]["EncDecAttention"]["k"]["kernel"] = (
+            t5lax_enc_dec_attention_key
+        )
+        flax_model.params["decoder"]["block"][str(layer_index)]["layer"]["1"]["EncDecAttention"]["o"]["kernel"] = (
+            t5lax_enc_dec_attention_out
+        )
+        flax_model.params["decoder"]["block"][str(layer_index)]["layer"]["1"]["EncDecAttention"]["q"]["kernel"] = (
+            t5lax_enc_dec_attention_query
+        )
+        flax_model.params["decoder"]["block"][str(layer_index)]["layer"]["1"]["EncDecAttention"]["v"]["kernel"] = (
+            t5lax_enc_dec_attention_value
+        )
+
+        flax_model.params["decoder"]["block"][str(layer_index)]["layer"]["1"]["layer_norm"]["weight"] = (
+            t5lax_cross_layer_norm
+        )
+
+        if split_mlp_wi:
+            flax_model.params["decoder"]["block"][str(layer_index)]["layer"]["2"]["DenseReluDense"]["wi_0"][
```

Comment:
```suggestion
            flax_model.params["decoder"]["block"][str(layer_index)]["layer"]["2"]["DenseReluDense"]["wi_0"][
```
we should define a utils like a `def flax_get_flatten` to which you give a key and it just does this!